.prettierrc.json

```diff
@@ -1,5 +1,7 @@
 {
-    "bracketSameLine": true,
-    "singleQuote": true,
-    "trailingComma": "all"
+  "bracketSpacing": false,
```

Comment:
I'm not sure about this one, it's a little ugly

Reply:
Wait, work in progress :D